src/solver/main_solver.cpp

```diff
@@ -40,18 +40,18 @@ int main()
     parse_json_file(tokens, orders, amms, Token::num_tokens, Token::idx_tokens, json_file);
 
     
-    // Just printing out the input
     std::cout.precision(std::numeric_limits<boost::multiprecision::mpf_float>::digits10);
 
-    for (auto &i: tokens)
-        print_token(i);
 
-    for (auto &i: orders)
-        print_order(i, tokens);
+    //for (auto &i: tokens)
```

Comment:
The best way to deal with this is to have some debugging mode and only print it in debugging. But this should be done in another pr